python/cugraph/cugraph/community/louvain.py

```diff
@@ -89,6 +93,14 @@ def louvain(G, max_level=None, max_iter=None, resolution=1.0, threshold=1e-7):
 
     """
 
+    isolated_vertices = list()
+    if is_nx_graph_type(type(G)):
+        num_of_vertices = len(G.nodes)
+        isolated_vertices = [v for v in range(num_of_vertices) if G.degree[v] == 0]
```

Comment:
this seems like a slow process vs putting this code later

G, isNx = ensure_cugraph_obj_for_nx(Gin)

if (isNx)
   isolated_vertices = (G.number_of_nodes == Gin.number_of_nodes())